backend/tests/models/test_partnership.py

```diff
@@ -0,0 +1,28 @@
+from bamboo.database import db, models
+
+
+def test_partnership(client):
+    profile = models.Media(path="test.png", content_type="image/png")
```

Comment:
```suggestion
    profile = models.Media.from_file("test.png")
```

Similar fixes apply for several occurrences